src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -23,6 +23,42 @@ public partial class Utf8JsonWriterTests
         private const int MaxEscapedTokenSize = 1_000_000_000;   // Max size for already escaped value.
         private const int MaxUnescapedTokenSize = MaxEscapedTokenSize / MaxExpansionFactorWhileEscaping;  // 166_666_666 bytes
 
+        private const string ExpectedIndentedCommentJsonOfArray = """
+/*Comment at start of doc*/
```

Comment:
Is there a test for writing comment twice in a row?

Reply:
Added.
There was already test case '[WriteCommentsInvalidTextAllowed](https://github.com/dotnet/runtime/blob/e1ddbca5a008b4b15e8e6b309cbab12ba8b9b842/src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs#L4154)' which wrote multiple comments in same line (so make sure behavior is not broken), but still added here for visible version. @krwq